packages/dds/tree/src/test/shared-tree/fuzz/topLevel.fuzz.spec.ts

```diff
@@ -91,6 +92,7 @@ describe("Fuzz - Top-Level", () => {
 			},
 			reconnectProbability: 0.1,
 			idCompressorFactory: deterministicIdCompressorFactory(0xdeadbeef),
+			skipMinimization: true,
```

Comment:
Why was this necessary?

Reply:
Whoops, it was for debugging, but removed now